lib/graphql/declaration.ts

```diff
@@ -50,6 +52,51 @@ const SUBMIT_DECLARATION = `
   }
 `;
 
+const GET_DECLARATION_STATS = `
+  query GetDeclarationStats($period: String!, $year: Int!) {
+    viewer {
+      mainAccount {
+        declarationStats(
+          period: $period,
+          year: $year
+        ) {
+          amount
+          uncategorized {
+            id
+            amount
+            name
+            purpose
+            valutaDate
+            userSelectedBookingDate
```

Comment:
Seems like this should be `selectedBookingDate` and  not `userSelectedBookingDate`
FE is complaining about it

Reply:
but it was already changed to selectedBookingDate🤔

Reply:
here https://github.com/kontist/js-sdk/pull/220/files

Reply:
ah! but not in the method itself 🤦‍♀️